scripts/common_test.py

```diff
@@ -50,7 +53,7 @@ class CommonTests(test_utils.GenericTestBase):
     """Test the methods which handle common functionalities."""
 
     @contextlib.contextmanager
-    def open_tcp_server_port(self):
+    def open_tcp_server_port(self) -> Iterator[int]:
```

Comment:
ditto as in _common.py_

Reply:
Hey, sorry for the trouble, I have created another PR for common.py and there I tried to make the changes you have suggested here. can you please check [this](https://github.com/oppia/oppia/pull/15258/) one and give your review. Thanks